src/render/texture.js

```diff
@@ -111,6 +113,10 @@ class Texture {
         return this.size[0] === this.size[1] && (Math.log(this.size[0]) / Math.LN2) % 1 === 0;
     }
 
+    get hashKey(): string {
+        return `${this.size[0]}.${this.premultiply ? 1 : 0}.${this.useMipmap ? 1 : 0}`;
```

Comment:
It should be safe not to care on UNPACK_PREMULTIPLY_ALPHA_WEBGL values when reusing texture allocations - in Texture.update value is set before texImage2D / texSubImage2D. It affects upload to texture.